spec.bs

```diff
@@ -2683,8 +2679,7 @@ An auction config is a [=struct=] with the following items:
   which reporting for this auction will agree on.
 
 : <dfn>per buyer currencies</dfn>
-:: An [=ordered map=] whose [=map/keys=] are [=origins=] and whose [=map/values=] are
-  [=currency tags=]. Specifies the currency bids returned by `generateBid()` or `scoreAd()` in
+:: Null or a {{Promise}} or failure or an [=ordered map=] whose [=map/keys=] are [=origins=] and whose [=map/values=] are [=currency tags=]. Specifies the currency bids returned by `generateBid()` or `scoreAd()` in

```

Comment:
Is there a difference between null and an empty map? If not, just use the empty map.

Reply:
How do I know what the initial value of this thing is?

Reply:
You need to say what the initial value is, either ", initially an empty map" or "The initial value is an empty map."

Reply:
Thanks, done.